README.md

```diff
@@ -12,13 +12,11 @@ uncompress("tests/fixtures/tree.tar.gz", dir.path(), Kind::TarGZip).unwrap();
 
 ## License
 
-Licensed under either of
+Licensed under either options at will:
 
  * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or https://www.apache.org/licenses/LICENSE-2.0)
  * MIT license ([LICENSE-MIT](LICENSE-MIT) or https://opensource.org/licenses/MIT)
 
-at your option.
-
```

Comment:
This is a license pattern followed by other Rust crates, I wouldn't start changing it here.